Quantum-Control-Applications-QuAM/Superconducting/quam_libs/components/gate_macros.py

```diff
@@ -55,31 +63,8 @@ def __post_init__(self) -> None:
         super().__post_init__()
         assert self.max_attempts > 0, "max_attempts must be greater than 0"
 
-    def apply(self, **kwargs) -> None:
-
-        pi_pulse: Pulse = (
-            self.pi_pulse
-            if isinstance(self.pi_pulse, Pulse)
-            else self.qubit.get_pulse(self.pi_pulse)
-        )
-        readout_pulse: ReadoutPulse = (
-            self.readout_pulse
-            if isinstance(self.readout_pulse, Pulse)
-            else self.qubit.get_pulse(self.readout_pulse)
-        )
-        if self.reset_type == "active":
-            from ..macros import active_reset
-            active_reset(self.qubit, pi_pulse_name=get_pulse_name(pi_pulse),
-                         readout_pulse_name=get_pulse_name(readout_pulse),
-                         max_attempts=self.max_attempts)
-            # self.qubit.reset_qubit_active(
-            #     pi_pulse_name=get_pulse_name(pi_pulse),
-            #     readout_pulse_name=get_pulse_name(readout_pulse),
-            #     max_attempts=self.max_attempts,
-            # )
-        else:
-            # Thermalize the qubit
-            self.qubit.wait(self.thermalize_time // 4)
+    def apply(self) -> None:
```

Comment:
The kwargs should still be part of the definition here for ensuring flexibility.